aws/scripts/install

```diff
@@ -191,17 +187,17 @@ update_configuration() {
 
 	installer_command_file="/var/www/html/installer/Command/InstallOnNewDatabaseCommand.php"
 
-  for line in {343..357}; do
-    docker exec orangehrm sed -i "343 d" $installer_command_file
+  for line in {342..356}; do
```

Comment:
will discuss this offline

Reply:
this needs to be changed. we can add arguments to installer like the upgrader. or we can take from env variables as Smanatha suggested https://github.com/orangehrm/orangehrm/issues/1776